frontend/src/features/idir/search/table/PermitSearchResultColumnDef.tsx

```diff
@@ -25,21 +29,23 @@ export const PermitSearchResultColumnDef = (
       const permit = props.row.original as PermitListItem;
       const { permitId, permitStatus, expiryDate, companyId } = permit;
 
+      const isPermitVoided = (permitStatus: PermitStatus) =>
```

Comment:
I think instead of creating this method, it would be more helpful to create a method that does the following:
```
const getDisplayedPermitStatus = (permitStatus: PermitStatus, expiryDate: Dayjs) => {
    if (permitStatus === PERMIT_STATUSES.VOIDED) {
        return PERMIT_STATUSES.VOIDED;
    }

    if (hasPermitExpired(expiryDate)) {
        return PERMIT_EXPIRED;
    }

    return permitStatus;
};
```